shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -515,3 +613,15 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
   return YES;
 }
 @end
+
+@implementation TouchTransparentView
```

Comment:
@amirh This is added recently to let touch events pass through the "clipping head" view when there are multiple platform views. (Tested with google map example: scrolling map). It wasn't in your last review, so I added this comment to make sure you notice the addition.

Reply:
f2f discussion: we're considering renaming this to a `ChildClippingView` and moving some of the clip logic inside. Open question to we want points outside to return NO from `pointInside`?

Reply:
done